ggshield/verticals/secret/output/secret_output_handler.py

```diff
@@ -20,18 +20,23 @@ def __init__(
         verbose: bool,
         output: Optional[Path] = None,
         ignore_known_secrets: bool = False,
+        ignore_removed_secrets: bool = True,
```

Comment:
Can you set this to False by default, to be consistent with the default value in SecretConfig?